docs/crab-home.md

```diff
@@ -5,22 +5,22 @@ sidebar_label: Crab Network
 slug: /
 ---
 
-Darwinia Crab Testnet (Crab for short) is a canary network with real economics for Darwinia, The positioning of Crab is similar to Polkadot's Kusama Network. Expect Chaos is a reasonable expectation.
+Darwinia Crab Testnet (Crab) is a canary network with real economic value for Darwinia, and its positioning is similar to Polkadot's Kusama Network. To Expect Chaos is a reasonable assumption!
 
-Crab mainly provides simulation environment for Darwinia's upgrade and application deployment, perform various radical experiments. Crab includes not only the complete software operation environment and network environment, but also the economic environment.
+Crab mainly serves as a testbed and simulation environment for Darwinia network upgrades and application deployment on which radical experiments can be performed. It not only provides a complete network and software execution environment, but also an economic environment.
 
-Crab is the first cross-chain NFT blockchain in Polkadot ecology, providing smart contract solutions, compatible with the Ethereum virtual machine, and supporting Dapps such as Defi and NFT to easily migrate to Polkadot.
+Crab is the first crosschain Bridge in the Polkadot ecosystem with Ethereum Virtual Machine (EVM) compatibility that natively supports DeFi dApps and NFTs, allowing them to  migrate seamlessly crosschain to Polkadot. 
```

Comment:
`crosschain` or `cross-chain`, which one is better? We are using `cross-chain` now.